### PR TITLE
Respect opendistro_security.auth.logout_url after logout

### PR DESCRIPTION
**Pull request by yuxitang-amzn**
*Issue #, if available:*

*Description of changes:*
Some IDP needs additional clear up step, which redirect to custom logout page. This change is to respect `opendistro_security.auth.logout_url` as it used to be.

Test:
* yarn tsc 
* yarn test:jest_ui public/apps/account

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by tianleh — APPROVED** [review]

File: public/apps/account/__snapshots__/account-nav-button.test.tsx.snap [hunk 1]
```diff
@@ -59,6 +59,7 @@ exports[`Account navigation button renders 1`] = `
               />
             </EuiListGroup>
             <EuiListGroupItem
+              color="subdued"
```
Comment (tianleh) [inline public/apps/account/__snapshots__/account-nav-button.test.tsx.snap:62]: Is this intentional? Looks unrelated to the topic of this change.

Comment (yuxitang-amzn) [reply public/apps/account/__snapshots__/account-nav-button.test.tsx.snap:62]: Yep this is to fix a snapshot test.
